enshrined/svg-sanitize/CVE-2022-23638.yaml

```
@@ -0,0 +1,8 @@
+title:     A cross-site scripting vulnerability
+link:      https://github.com/darylldoyle/svg-sanitizer/releases/tag/0.15.0
```

```suggestion
link:      https://github.com/advisories/GHSA-fqx8-v33p-4qcc
```

Sorry one more thing, link should go to security announcement, not tag for the fix